common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -93,13 +93,13 @@ function (Iterator) {
          * initial configuration.
          * @param {array} speeds List of speeds available for the player.
          */
-        render: function (speeds) {
+        render: function (speeds, speed) {
```

Comment:
Can we add js test to check if the expected speed option get highlighted and being set.